build_dynamic_prompt.py

```diff
@@ -1,9 +1,9 @@
 import random
 import re
-from csv_reader import *
-from random_functions import *
-from one_button_presets import OneButtonPresets
-from superprompter.superprompter import *
+from .csv_reader import *

```

Comment:
Hi! Writing it like this will result in errors in A1111 (or running my test scripts locally):

_"ImportError: attempted relative import with no known parent package"_

I can fix it by removing the . in these files, and keeping them in the __init__.py and OneButtonPromptNodes.py

However, I am not a professional programmer. This might not be enough to create a fix for ComfyUI manager.

Is there a way to have it both?

Reply:
It seems to be because the structure for executing modules in A1111 and ComfyUI is different.
There is a way to do it, but I'll test it in the A1111 environment later and submit a PR again.

Reply:
Thank you, TIL you can do it like this.

This works perfectly. Merging the changes.